x11/urw-fonts/Portfile

```diff
@@ -1,51 +1,59 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-
-name            urw-fonts
-version         1.0.7pre44
-categories      x11 fonts
-maintainers     nomaintainer
-license             GPL-2+
-description     Cyrillized free URW fonts
-long_description \
-                These fonts were made from the free URW fonts distributed with ghostcript. \
-                There are NO changes in the latin part of them (I hope). \
-                Cyrillic glyphs were added by copying suitable latin ones \
-                and painting oulines of unique cyrillic glyphs in same style as the others.
-platforms       darwin
-supported_archs noarch
-homepage        ftp://ftp.gnome.ru/fonts/urw/release/
-master_sites    macports \
-                ftp://ftp.gnome.ru/fonts/urw/release/
-use_bzip2       yes
-checksums       md5 51c6c2690593cd9bd92f197a6f2ff8bd \
-                sha1 465fd5faab9ba9e7590f1bf761015862eb09748d \
-                rmd160 52bddb1e267e6aa3cace9197293718e4919384bd
-
-depends_run     port:fontconfig
-
-extract.mkdir   yes
-
-use_configure   no
+PortSystem          1.0
+
+name                urw-fonts
+version             2017-07-27
+revision            1
+epoch               1
```

Comment:
Both the `revision` and the `epoch` lines need to be removed. "2017-07-27" will appear to be later than "1.0.7pre44".

Reply:
Agreed. That said, is 2017-07-27 what the upstream calls this version?

Reply:
Will remove `revision` and `epoch`, and upstream doesn't set any tag for this commit – it's simply the date mentioned in the [commit](http://git.ghostscript.com/?p=urw-core35-fonts.git;a=log;h=HEAD).

Reply:
@mocja Thanks for finding that!  Will set up the portfile to fetch the tarball.

Reply:
I see they have no real numbering scheme, so I suppose the date will have to do.

Reply:
By looking at http://git.ghostscript.com/?p=urw-core35-fonts.git;a=summary I don't see any super usable version information. Individual fonts have their own versions listed. Commit 149a20b9dd7cc2b416d91540355f3f6f50b9d2d6 mentions version 1.10 and then all the fonts in fact bear the version number 1.10.

Commit 24de0b9eb8fdb96b2446fe24c17b537a947cec22 changes some font versions to version 3 (I did not check how many fonts have this version modified). But then at some point all the versions were set back to 1.00 (some are still at 2.00). Beats me.

Reply:
Yah, date's all we've got.